recipes/gumbo-parser/all/conanfile.py

```diff
@@ -1,69 +1,89 @@
-from conans import ConanFile, AutoToolsBuildEnvironment, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc, unix_path
 import os
 
+required_conan_version = ">=1.53.0"
+
 
 class GumboParserConan(ConanFile):
     name = "gumbo-parser"
```

Comment:
#14313 was it this one that say "unvedor"

Reply:
indeed, litehtml recipe should be updated to depend on gumbo-parser.